Automattic/jetpack
# Boost: Add a notice for fresh start

Fixes Automattic/boost-cloud#363

## Proposed changes:

* A notice to display when there is no pre-existing data

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Start with a new boost site
* Upgrade to premium
* See the performance history panel to find the notice.

![image](https://github.com/Automattic/jetpack/assets/3737780/f055df51-95b5-4b3c-a0be-452a8d970a2f)
